subsys/net/lib/lwm2m/lwm2m_rw_oma_tlv.c

```diff
@@ -903,6 +903,53 @@ static int write_tlv_resource(struct lwm2m_message *msg, struct oma_tlv *tlv)
 	return 0;
 }
 
+static int lwm2m_multi_resource_tlv_parse(struct lwm2m_message *msg,
+					  struct oma_tlv *multi_resource_tlv)
+{
+	struct oma_tlv tlv_resource_instance;
+	int len2;
+	int pos = 0;
+	int ret;
+
+	if (msg->in.block_ctx) {
+		msg->in.block_ctx->res_id = multi_resource_tlv->id;
+	}
+
+	if (multi_resource_tlv->length == 0U) {
+		/* Mo nested data Create path level 3  */
```

Comment:
I don't understand this comment?
What are you trying to refer here? If there is not nested data, do you mean that we create empty resources, or what? What does path level 3 refer to?
Can this be like `/* No data for resource instances, so create only a resource  */`

Reply:
Ok I update comment by your proposal.